apps/github/src/events/issues-opened.ts

```diff
@@ -0,0 +1,20 @@
+import Event from '../libs/event';
+import AskBuddieBot from 'src/libs/askbuddiebot';
+
+class IssuesOpened implements Event {
+    name = 'issues.opened';
+
+    async handleEvent(payload: Payload): Promise<boolean> {
+        const issue = payload.issue as Issue;
+
+        console.info(`Issue opened: ${issue.title}`);
+
+        const body = issue.body + `<br/><br/>Ref: ${issue.id}`;
```

Comment:
Please prefix the issue id with the repository name `<REPO_NAME>-<ID>`

Reply:
okay.